test/dragAndDrop.test.js

```diff
@@ -0,0 +1,34 @@
+import dragAndDrop from './__mocks__/dragAndDropMock';
+import './__mocks__/localStorageMock';
+
+describe('dragAndDrop', () => {
+  it('should throw an error if no task is given', () => {
+    expect(() => {
+      dragAndDrop.editTask();
+    }).toThrow();
+  });
+
+  it('should throw an error if no item is given', () => {
+    expect(() => {
+      dragAndDrop.clearCompleted();
+    }).toThrow();
+  });
+
```

Comment:
- [ ] Kindly add tests to check for the clear completed method. Good job on the edge cases too 👍🏾